README.md

```diff
@@ -1,20 +1,30 @@
 # Tracker-COVID-19
-Project Description
+This React capstone project is about building a Mobile Web Application to check a list of metrics (numeric values) that I will create making use of React and Redux. Thanks to the COVID-19 metrics I was able to fetch data by using Fetch API, which helped me to get the all the stats per country needed for the project.
```

Comment:
- [ ] Please, consider, that as I was requested before by the previous reviewer, you did not upload the video. I could not find it Readme file and also in the PR description. Please, if you record the video, put it in your Readme file and ask for the next review 👍